_includes/footer.html

```diff
@@ -25,7 +25,16 @@ <h5 class="white-text">viperdev</h5>
       </div>
       <div class="col l4 offset-l2 s12">
         <h5 class="white-text">Any Questions or Queries ?</h5>
-        <a href="mailto:contact@viperdev.io" class="mailbtn btn-large waves-effect grey-text text-lighten-4">Drop us a <span class="mail-icon"> <i class="mailicn material-icons">email</i></span></a>
+
+<div >
+ <a href="https://gitter.im/viper-development/support" target="blank" class="mailbtn btn-large waves-effect grey-text text-lighten-4">CHAT WITH US <span class=""><i class="mailicn fa fa-commenting" aria-hidden="true"></i>
```

Comment:
why is this all caps? Also can't you use https://sidecar.gitter.im/ to embed the chat?